examples/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
-add_subdirectory(component)
-add_subdirectory(dom)
-add_subdirectory(util)
+if(FTXUI_BUILD_EXAMPLES)
```

Comment:
What about moving this in the main CMakeLists.txt. Wrapping "add_subdirectory(examples")

Reply:
Clearly a better solution if you plan to use the examples/CMakeLists.txt only for build the examples. If that's the case I'm more than happy to make the changes.

Reply:
I was thinking about doing:
~~~cmake
if (FTXUI_BUILD_EXAMPLES)
  add_subdirectory(examples)
endif()
~~~

I am not sure to understand fully what you meant in you previous response.

Reply:
In this way the file in examples is not included, if its sole purpose is to build the examples it is better to use the approach proposed by you, if instead the file will have other functions it is better that the controls are in the file itself. So it depends on what are your intentions with the file

Reply:
I see!
The goal is to build the examples only. So I would prefer adding the "if" in the around the add_subdirectory.